### ISSUE TITLE
Compass deployment, mysql data not synchronized

### ISSUE THREAD

**Issue by GikiSea** [cubefs/compass#30]
Please help take a look at this issue，i find the data of dolphinscheduler is not synced to compass(except the success tasks in task_instance)

![mysql-compass](https://user-images.githubusercontent.com/119785792/233962996-32ae8693-49f2-4155-bbd6-4f3fe943fea3.jpg)
![mysql-compass2](https://user-images.githubusercontent.com/119785792/233963006-8c6f62e8-e54d-416f-a6e4-fd56f39ff100.jpg)
![mysql-compass3](https://user-images.githubusercontent.com/119785792/233963014-9c19848e-0919-4655-9e05-c25e4dfd9197.jpg)
![mysql-dolphine](https://user-images.githubusercontent.com/119785792/233963016-71fdcefa-eb8a-4df3-a0da-166bb497545b.jpg)

**Comment by nilnon** [comment]
@GikiSea You need to check whether other tables have incremental data. If you need to fully synchronize historical data, you need to truncate the task_syncer_init table and restart task-syncer. In addition, you have not created the dolphinscheduler dependent table, please refer to the documentation.

**Comment by nilnon** [comment]
@GikiSea You need to observe the log of the task-canal-adapter module.

**Comment by GikiSea** [comment]
@nilnon  Thanks, it works now. And i found task-application print error when i run a task on dolphinscheduler:
![task-application-error](https://user-images.githubusercontent.com/119785792/234750434-99475155-92ad-4076-8396-576107eaf583.png)
![task-application-warn](https://user-images.githubusercontent.com/119785792/234750439-70d75d34-1f03-4870-b483-71a5933bb3a2.png)
the class in error logs is in hadoop-hdfs-client-3.3.4.jar, and the cluster's hadoop version i used is 3.2.1, could this be the problem?
![lib](https://user-images.githubusercontent.com/119785792/234751561-828baddf-fb2f-4b6d-8523-c1539e6b4f9d.jpg)

**Comment by nilnon** [comment]
@GikiSea Please recompile with the latest version, we optimized the configuration settings of this module. The above problem is due to the inconsistent nameservice configuration of application-hadoop.yml and application-dolphinscheduler.yml.

**Comment by GikiSea** [comment]
@nilnon i had use the same config to deploy the latest version, but task-cannal-adapter print error logs now:
![task-cannal-adapter](https://user-images.githubusercontent.com/119785792/234794937-c5ee36db-a7a8-4337-abfb-ccb91e4ca818.jpg)

**Comment by nilnon** [comment]
Have you recreated the table? Please check which table in compass mysql lacks the restart_time field. This table structure needs to be the same as the dolphinscheduler table structure.

**Comment by GikiSea** [comment]
Actually, i had try recreated the tables. But it still prints error logs, and i check the not-matched columns(like restart_time or task_execute_type) in compass.sql and dolphinscheduler.sql, seems there're not such columns in compass tables?I look back at yesterday's, find the same errors show at first, but it looks like that this problem just turn well without any action yesterday.It makes me confused...

**Comment by GikiSea** [comment]
Thanks, i have create the right structure table.Does task-application depends on the logs i collect  to operation? Task-application still prints the same errors and web show nothing(i haven't deploy flume to collect dolphin logs)

**Comment by GikiSea** [comment]
Thanks, i have create the right structure table(i use dolphinscheduler v2.0.7).Does task-application depends on the logs i collect  to operation? Task-application still prints the same errors and web show nothing(i haven't deploy flume to collect dolphin logs)

**Comment by nilnon** [comment]
@GikiSea Yes, task-applicaiton depends on task running logs, which need to be collected and uploaded to hdfs. Does your hadoop cluster enable kerberos? If so, has the relevant kerberos information been configured according to the latest applicaiton-hadoop.yml?

**Comment by GikiSea** [comment]
@GikiSea No, i used to deploy the early version compass which doesn't support kerberos, so i turn off the cluster authentication

**Comment by nilnon** [comment]
@GikiSea Because I don't know the configuration information of application-hadoop.yml and application-dolphinscheduler.yml, so I can't judge the specific reason.
